backend/app/models/ScheduleState.py

```diff
@@ -0,0 +1,9 @@
+from sqlalchemy import Column, Integer, String
+
+from .Base import Base
+
+
+class ScheduleState(Base):
```

Comment:
Let's rename this to `ScheduleStatus`, I think this would be more indicative of a progession since `Schedule` is general.